### DISCUSSION TITLE
Config option for disablying Elytra + Armor

### DISCUSSION THREAD

**Discussion by Jous4n** [BeansGalaxy/Beans-Backpacks-2#150] [Feedback] [answered]
I think it'd be cool to have a way to disable via config the possibility of using Elytra and Armor at the same time. Some players/modpacks could consider this feature a bit overpowered, specially for survival worlds.

**Comment by BeansGalaxy** [comment, marked as answer]
By removing the elytra from the "elytra_items" in config returns it to normal. Update to the 20.1-0.30-v2 first because I just added config menus and I wanna show it off. :)

**Reply by Jous4n** [reply]
:O Works perfectly fine! Thanks for this addition!

**Reply by Jous4n** [reply]
Btw, is there a way to also do the same to Winged Backpacks? Because you can just bypass the elytra back config by just using one of those.

https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/118950641/b867b10c-0ac3-4b21-8b3e-565bd5f05f42

**Reply by Jous4n** [reply]
I think it would also be required to disable the Winged Backpack recipe for this condition, so that players won't waste their Elytra.
